openmls/src/group/public_group/mod.rs

```diff
@@ -0,0 +1,265 @@
+use openmls_traits::{crypto::OpenMlsCrypto, types::Ciphersuite, OpenMlsCryptoProvider};
```

Comment:
Can you add some doc comments to the module here explaining what it's about and how it works?